addons/medical/Language/ACE_Medical_localization.st

```diff
@@ -11,6 +11,13 @@ StringTable {
    Author "kexanone"
    LastChanged "apollo-jk"
   }
+  CustomStringTableItem "{61D51487241F1A28}" {
+   Id "ACE_Medical-FailReason_CriticallyInjured"
```

Comment:
Make the key more consistent with the localization

```suggestion
   Id "ACE_Medical-FailReason_TooInjured_MedicalFacilityRequired"
```

Reply:
Renamed the localization variable and the local variable it fills with a name more consistent with the localization name.